.travis.yml

```diff
@@ -40,7 +40,7 @@ before_script:
   - mypy syft || echo "Type errors found, continuing build..."
 script:
   - coverage run -m pytest test
-  - coverage report --omit="syft/grid/*" --ignore-errors --fail-under 95 -m
+  - coverage report --omit="syft/grid/*","syft/frameworks/torch/nn/rnn.py" --ignore-errors --fail-under 95 -m
```

Comment:
Needed to make the rest of the PR pass coverage requirements?

Reply:
Yes - it looks like because there were no tests in that directory at all that it wasn't being included in code coverage. When I added tests for some things in the directory, it then had to fail for the rest. It's out of scope for this PR to test that rnn right now, but we can get to it eventually.

Reply:
created an issue to capture this

https://github.com/OpenMined/PySyft/issues/2926